.github/ISSUE_TEMPLATE/new_bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels: 
+assignees: '@abbylindsay'
+
+---
+
+**Describe the bug- why is it wrong?:**
+<!-- A clear and concise description of what you/the participant is seeing and why it is a problem. -->
+
+**How was this bug identified:**
+<!-- Who saw it? What was the context? -->
+
+**How to reproduce?:**
```

Comment:
We should make fields required as appropriate. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Reply:
at first glance it looks like the issue templates would have to be added to each repo to do this... is that worth it? ill look at it more